packages/env/src/constants/directus-variables.ts

```diff
@@ -19,6 +19,7 @@ export const DIRECTUS_VARIABLES = [
 	'GRAPHQL_QUERY_TOKEN_LIMIT',
 	'MAX_BATCH_MUTATION',
 	'LOGGER_.+',
+	'QUERY_MAX_DEPTH',
```

Comment:
This can be a bit confusing due to `QUERY_LIMIT_MAX` and `QUERY_LIMIT_DEFAULT`, should we rename it to something more unique?

```suggestion
	'QUERYSTRING_PARSE_MAX_DEPTH',
```

Reply:
Yes we should! Though I'd swap the sequence to be consistent with other envs such as `MAX_RELATIONAL_DEPTH` and `PRESSURE_LIMITER_MAX_EVENT_LOOP_UTILIZATION`

```suggestion
	'QUERYSTRING_MAX_PARSE_DEPTH',
```